dev-cpp/notcurses/notcurses-2.2.2.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_ECLASS="cmake"
+inherit cmake-multilib
+
+DESCRIPTION="Blingful TUIs and character graphics"
+HOMEPAGE="https://notcurses.com"
+SRC_URI="https://github.com/dankamongmen/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+DEPEND="dev-libs/libunistring
+	media-video/ffmpeg
+	>=sys-libs/readline-8.0"
+RDEPEND="${DEPEND}"
+BDEPEND="doc? ( app-text/pandoc )"
+
+src_prepare() {
+	cmake_src_prepare
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DUSE_DOCTEST=OFF
+		-DUSE_QRCODEGEN=OFF
+		-DUSE_PANDOC="$(usex doc)"
+	)
+	cmake-multilib_src_configure
```

Comment:
```
if(${USE_FFMPEG})
pkg_check_modules(AVCODEC REQUIRED libavcodec>=57.0)
pkg_check_modules(AVFORMAT REQUIRED libavformat>=57.0)
pkg_check_modules(AVUTIL REQUIRED libavutil>=56.0)
pkg_check_modules(SWSCALE REQUIRED libswscale>=5.0)
set_property(GLOBAL APPEND PROPERTY PACKAGES_FOUND FFMpeg)
elseif(${USE_OIIO})
pkg_check_modules(OIIO REQUIRED OpenImageIO>=2.1)
set_property(GLOBAL APPEND PROPERTY PACKAGES_FOUND OpenImageIO)
```
You could make it visible here that ffmpeg is the chosen configure flag, but that openimageio is also an option. Gentoo users do love the choice ;) but imho you are doing correctly just choosing ffmpeg without any ugly `REQUIRED_USE` restrictions.

Reply:
Sorry, but I'm not sure if/what you're asking me to do here :/.

Reply:
I basically wanted you to make the configure options visible, similarily to how 
```
	-DUSE_DOCTEST=OFF
		-DUSE_QRCODEGEN=OFF
		-DUSE_STATIC=OFF
```
is currently visible in your ebuild. So people know about these configure options. We have `EXTRA_ECONF` available to control "hard-coded" variables for more user choice.

Reply:
Done.